packages/card/card.tsx

```diff
@@ -0,0 +1,151 @@
+import classNames from 'classnames'
+import Image from 'next/image'
+import Link from 'next/link'
+import ProductBadge from '@hashicorp/react-product-badge'
+import { IconArrowRight24 } from '@hashicorp/flight-icons/svg-react/arrow-right-24'
+import type {
+  CardProps,
+  ThumbnailProps,
+  MetaProps,
+  ContentProps,
+  HeadingProps,
+  ProductBadgesProps,
+  DescriptionProps,
+} from './types'
+import s from './style.module.css'
+
+function Card(props: CardProps) {
+  const {
+    appearance = 'light',
+    withArrow = true,
+    hoverRef,
```

Comment:
Should we be forwarding refs here? https://reactjs.org/docs/forwarding-refs.html

Reply:
Ah, yes we probably should be! It seems this component is a good use case for it.

Reply:
Addressed here, need to sort out TS typing now that the component is using `forwardRef` 🤔 
https://github.com/hashicorp/react-components/pull/776/commits/94ec2dd71a975df57df5767df3f77619274cc9be